README.md

```diff
@@ -26,6 +26,54 @@ Prerequisites:
 Installation (make)
 ------------
 
+### Installing ls1-MarDyn using cmake
+This is the recommended way of building ls1-MarDyn.
+
+#### Configuration
+To build ls1-MarDyn using cmake first create an additional directory on the root ls1-MarDyn directory and change into that directory.
+```bash
+mkdir build
+cd build
+```
+Next, `cmake` has to be executed. In most cases, you will have to specify the compiler with which ls1-MarDyn should be built:
+```bash
+CC=clang CXX=clang++ cmake ..
+```
+Some of the most common compilers are
+| Compiler      | CC    | CXX     |
+|---------------|-------|---------|
+| GCC           | `gcc`   | `g++`     |
+| Intel Classic | `icc`   | `icpc`    |
```

Comment:
I'd rather throw this one out because it can't compile AutoPas due to "internal compiler errors".

Reply:
I have added a "Quick guide" to be more focused on one way and to provide a fast instruction. However, I would keep the detailed instructions and the different options for the "interested reader". Furthermore, trying different compilers can speed-up the code at almost no cost.

Reply:
One thing we can consider is to create a wiki page with the detailed instructions, and keep the readme concise. Then we could add even more build instructions in the wiki page without worry, such as for fugaku, hawk etc.

Reply:
In general, this is a good idea!
I personally, don't like the doxygen wiki much (maybe due to my lack of knowledge):
- Why is the main page the "Droplet Coalescence" example?
- Where is the "Visualization with MegaMol" page? I don't really get, how the .dox files are connected to the Doxygen pages. [This pages list](https://ls1mardyn.github.io/ls1-mardyn/pages.html) does not seem to be complete.

Reply:
PS: It's kind of funny (or sad) to see the [todo page](https://ls1mardyn.github.io/ls1-mardyn/todo.html) nobody cares about 😆

Reply:
This page is automatically generated by code comments. But yes, the most important stuff should be handled via GitHub issues.